src/sdk-reference/js/6/extend-sdk/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: sdk.html.hbs
+title: Extends the SDK
+description: Extends the SDK
+order: 410
+---
+
+# Extends the SDK with a custom SDK controller
+
+It is possible to extend the SDK's functionality by adding new controllers.  
+
+These controllers generally correspond to [custom controllers created in a plugin]({{ site_base_path }}plugins/1/essentials/controllers) but not necessarily. Thus, it is possible to use the actions of its plugin in the SDK in the same way as the other actions of the Kuzzle API.  
+
+## Define a custom SDK controller
+
+A custom SDK controller is a class inheriting from the [BaseController]({{ site_base_path }}sdk-reference/js/6/base-controller) class and defining methods that correspond to API actions.  
+
+This class is exposed alongside the other class of the SDK module.  
+
+This controller can then be added to the SDK with the [Kuzzle.useController]({{ site_base_path }}sdk-reference/js/6/kuzzle/use-controller) method.
```

Comment:
Well... it must be added _before_ it is "exposed", so this sentence should come before the previous one (or it can be removed if you use one of my proposals above, it's your call)